### PR TITLE
Fix empty appimage

### PR DESCRIPTION
**Pull request by M0ses**
* do not break if build section doesn`t exists in yml
* various changes for pylint

### CODE REVIEW

File: TarSCM/tasks.py [hunk 1]
```diff
@@ -119,7 +126,6 @@ def _process_single_task(self, args):
         # self.scm_object is need to unlock cache in cleanup
         # if exception occurs
         self.scm_object = scm_object   = scm_class(args, self)
-        helpers      = scm_object.helpers
```
Comment (aspiers) [inline TarSCM/tasks.py]: Technically this removal should be in a separate commit.

**Comment by aspiers** [comment]
Just expanding a bit on my request for greater clarity:

- The PR description says "do not break if build section doesn`t exists in yml"
- The corresponding commit message says "fix for empty appimage.yml"

So it's not clear to me whether this change is targeting a missing appimage.yml file, or a missing build section within a .yml file.  This is probably because of my ignorance in this area: I've never worked with AppImages at all, let alone building AppImages in OBS.  Hopefully that serves as an example of why a more comprehensive commit message as described in that presentation by @cmurphy is worth the extra 30 seconds to write.

**Comment by M0ses** [comment]
@aspiers Thanks for your feedback - I tried to improve #160 with your suggestions. I will close this PR in favour to #160
